src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * Math.pow(getRadius(), 2);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea()
+                + ", radius: " + getRadius()
+                + ", color: " + getColor());
+    }
```

Comment:
```suggestion
    public double getArea() {
        return Math.PI * Math.pow(radius, 2);
    }

    @Override
    public void draw() {
        System.out.println("Figure: circle, area: " + getArea()
                + ", radius: " + radius
                + ", color: " + getColor());
    }
```
it is a good practice not to use getters when you have direct access to a field